test/form/samples/export-default-import/_expected/es.js

```diff
@@ -1,5 +1,2 @@
 import x from 'x';
```

Comment:
Yes exactly, if we could track per-chunk usage then this can change, but until then the import statements are never pruned. These statements are well-merged though so it typically doesn't result in any bloat.